R/big_n.R

```diff
@@ -0,0 +1,104 @@
+#' Big N Structure
+#'
+#' Big N strucutre allows you to specify which values should become the big n
+#' values and how they should be formatted. Values are specified by providing
+#' the row value(s) of the parameter column for which the values are big n's.
+#' This will remove these from the body of the table and place them into columns
+#' matching the values in the column column(s). The default formatting is `N = xx`,
+#' on its own line, but that can be changed by providing a different `frmt()` to
+#' `n_frmt`
+#'
+#' @param param_val row value(s) of the parameter column for which the values
+#'   are big n's
+#' @param n_frmt [frmt()] to control the formatting of the big n's
+#'
+#' @return big_n_structure object
+#' @export
+#'
+big_n_structure<- function(param_val, n_frmt = frmt("\nN = xx")){
+  structure(
+    list(
+      param_val = param_val,
+      n_frmt = n_frmt
+    ),
+    class = c("big_n_structure","structure")
+  )
+}
+
+
+
+apply_big_n_df <- function(col_plan_vars, columns, value, big_n_df){
```

Comment:
As we discussed over chat, this code expects `col_plan_vars` to be `vars`. But if no col_plan is supplied to tfrmt, then `col_plan_vars` ends up being a chr vector. @thebioengineer suggested an edit below

Reply:
It also would be good to add roxygen for this function